server/sonar-server/src/main/java/org/sonar/server/ws/KeyExamples.java

```diff
@@ -31,6 +31,9 @@
 
   public static final String KEY_BRANCH_EXAMPLE_001 = "feature/my_branch";
 
+  public static final String NAME_WEBHOOK_EXAMPLE_001 = "my-webhook";
```

Comment:
As whitespaces are allowed, I suggest to use value "My Webhook".

Reply:
done